CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Next release
 
+* Fix a bug on re-generating binstub (#534)
```

Comment:
This message describes perfectly this changes. Thanks so much. I'll change this.